src/utils/index.ts

```diff
@@ -0,0 +1,8 @@
+export {buildCommandArgs} from './buildCommandArgs';
```

Comment:
You can prevent some boilerplate with

```ts
export * from './buildCommandArgs';
// etc.
```

Reply:
I actually prefer making the export explicit, in case we want to add private helper functions inside the util files, or common helpers to all utils which we don't want to mistakingly export by default. WDYT?

Reply:
Private helper functions wouldn't be exported so they will not pollute the index module, and I guess common helpers would go in their own files? 😉